### PR TITLE
Add TypeScript 'abstract classes' guide.

### PR DESCRIPTION
**Pull request by jasmeetsohal**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.

Closes #XXXXX

### CODE REVIEW

**Comment by jasmeetsohal** [comment]
I have added content on  Abstract Classes for TypeScript guide could you please review @raisedadead Thanks

File: guide/english/typescript/abstract-classes/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: Abstract Classes
+---
+
+# Abstract Classes
+
+Abstract classes are super classes which can be derived by other classes.The class marked with `abstract` keyword
```
Comment (moT01) [inline guide/english/typescript/abstract-classes/index.md]:
```suggestion
Abstract classes are super classes which can be derived by other classes. The class marked with the `abstract` keyword
```

File: guide/english/typescript/abstract-classes/index.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+```typescript
+abstract class FCC{
+    abstract makeSkill(): void;
+    getSound():void{
+        console.log("Coding...");
+    }
+}
+```
+
+Methods enclosed to abstract class which are marked `abstract` they do not contain method body and must be implemented in derived
```
Comment (moT01) [inline guide/english/typescript/abstract-classes/index.md]:
```suggestion
Methods enclosed to abstract class which are marked `abstract` do not contain a method body and must be implemented in the derived
```

**Review by moT01 — CHANGES_REQUESTED** [review]
Looks real good @jasmeetsohal - I made a few grammar change suggestions. If you can change these, we can get this merged. Thanks.

**Review by moT01 — APPROVED** [review]
Looks good @jasmeetsohal - thanks for contributing to freeCodeCamp! 🎉

**Pull request merged**